WebWasted/Controllers/FoodsController.cs

```diff
@@ -67,12 +98,16 @@ public IEnumerable<Food> Get(int id)
         public IActionResult Post([FromBody] NewFoodArguments args)
         {
             Food food = null;
+            
+            
             switch (args.type)
             {
                 case 1: // food obj is weighed
                     try
                     {
                         food = new WeighedFood(args.owner, args.name, args.description, args.fullPrice, args.foodType, args.amount, args.expTime);
+                     
+                        food.PhotoFileName=args.fileName;
```

Comment:
Can a food offer not have a photo ? Maybe we could check if it has it ? 
Also if it has to have a photo, then maybe it is a better idea to put this argument into the constructor, so that it gets passed even if we create a food class somewhere else (otherwise we can forget to add the photo "manually" so to speak)

Reply:
When photo is not attached to the offer, it is automatically set to "anonymous.jpg" (88 line of this code).
Instead of throwing an exception, we process it immediately